google-cloud-spanner/src/main/java/com/google/cloud/spanner/DatabaseClient.java

```diff
@@ -191,6 +193,77 @@ CommitResponse writeWithOptions(Iterable<Mutation> mutations, TransactionOption.
   CommitResponse writeAtLeastOnceWithOptions(
       Iterable<Mutation> mutations, TransactionOption... options) throws SpannerException;
 
+  /**
+   * Applies batch of mutation groups in a collection of efficient transactions. The mutation groups
+   * are applied non-atomically in an unspecified order and thus, they must be independent of each
+   * other. Partial failure is possible, i.e., some mutation groups may have been applied
+   * successfully, while some may have failed. The results of individual batches are streamed into
+   * the response as and when the batches are applied.
+   *
+   * <p>Since this method does not feature replay protection, it may attempt to apply {@code
+   * mutation groups} more than once; if the mutation groups are not idempotent, this may lead to a
+   * failure being reported when the mutation group was applied once. For example, an insert may
```

Comment:
The current text seems a bit strange. Should it have been this?

```suggestion
   * failure being reported when the mutation group was applied more than once. For example, an insert may
```

Reply:
Can we change this to something like: 'The mutation groups can be applied more than once. This can lead to failures if the mutation groups are non-idempotent. For example, an insert that is replayed can return an `AlreadyExists` error.'

I assume that the intended meaning of the word 'once' in the current text '...failure being reported when the mutation group was applied once.' is something like 'it had already been applied once by an earlier attempt'. But that is not how I read it as a new reader. I read it as 'It will return `AlreadyExists` if it is applied excatly one time', which did not make sense to me.

Reply:
Makes sense, made changes to doc.